dff/script/core/message.py

```diff
@@ -35,10 +37,20 @@ class Command(DataModel):
     a command that can be executed in response to a user input.
     """
 
-    ...
+    pass
+
+
+class Attachment(DataModel):
```

Comment:
I think we should list all messenger interfaces that support a specific attachment.
This could be done in two ways:
1. In the docstring of each class.
2. By adding a field to the attachment model that contains a list of all supported messenger interface types.

I think the latter is preferable.